Gopkg.lock

```diff
@@ -926,7 +926,7 @@
   version = "kubernetes-1.13.1"
```

Comment:
all vendor changes does not belong to this commit

Reply:
moved to different commit